docs/source/examples/auto-failover.rst

```diff
@@ -3,6 +3,17 @@
 Auto-provisioning GPUs
 ==========================
 
+.. tip::
+
+  No action is required to use the auto-failover feature.
```

Comment:
This is the first time `auto-failover` is mentioned (the title and sidebar states `Auto-provisioning GPUs`. This tip felt a little discontinuous and I was thinking what is auto-failover and whether I clicked on the right link.

Should we change the mentions of `auto-failover` to `auto-provisioning`? 
```
No action is required to use the auto-provisioning feature.
```